app/Http/Requests/UpdateAdvertApplicationRequest.php

```diff
@@ -24,7 +24,7 @@ public function authorize()
     public function rules()
     {
         return [
-            //
+            'content' => 'string',
```

Comment:
You are missing some validation rules here, the content needs:
- to be required
- to be filled
- to have a maximum size